units/units.cpp

```diff
@@ -144,12 +144,14 @@ static const umap base_unit_names{
                                        // 1000 m^3 not mega meters cubed
     {kg, "kg"},
     {mol, "mol"},
+    {unit(0.1, m), "dm"},  // don't want to use deci in most conversion contexts
```

Comment:
I believe `dL` or `dl` are also quite common? Also, I don't know if you support decibel, but if you do, there is `dB`.

Reply:
I will add a `dl` exception.  dB is handled differently since that is an equation unit